test/interpreter/function-roman.spec.ts

```diff
@@ -0,0 +1,82 @@
+import {ErrorType, HyperFormula} from '../../src'
+import {ErrorMessage} from '../../src/error-message'
+import {adr, detailedError} from '../testUtils'
+
+describe('Function ROMAN', () => {
+  it('should return #NA! error with the wrong number of arguments', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=ROMAN()', '=ROMAN(1, 1, 1)'],
+    ])
+
+    expect(engine.getCellValue(adr('A1'))).toEqualError(detailedError(ErrorType.NA, ErrorMessage.WrongArgNumber))
+    expect(engine.getCellValue(adr('B1'))).toEqualError(detailedError(ErrorType.NA, ErrorMessage.WrongArgNumber))
+  })
+
+  it('should properly truncate values and use defaults', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=ROMAN(499)'],
+      ['=ROMAN(499, TRUE())'],
+      ['=ROMAN(499, FALSE())'],
+      ['=ROMAN(499.9)'],
+      ['=ROMAN(499, 1.1)'],
+    ])
+    expect(engine.getCellValue(adr('A1'))).toEqual('CDXCIX')
+    expect(engine.getCellValue(adr('A2'))).toEqual('ID')
+    expect(engine.getCellValue(adr('A3'))).toEqual('CDXCIX')
+    expect(engine.getCellValue(adr('A4'))).toEqual('CDXCIX')
+    expect(engine.getCellValue(adr('A5'))).toEqual('LDVLIV')
+  })
+
+  it('should throw correct error', () => {
```

Comment:
IMO small improvement:
```suggestion
  it('should throw correct error if arguments are out of bounds', () => {
```